packages/core/src/shared/telemetry/util.ts

```diff
@@ -292,12 +298,16 @@ export async function getComputeEnvType(): Promise<EnvType> {
             return 'ec2-amzn'
         }
         return 'ec2'
-    } else if (env.remoteName) {
+    } else if (env.remoteName === 'wsl') {
         return 'wsl'
     } else if (isAutomation()) {
         return 'test'
     } else if (!env.remoteName) {
```

Comment:
`env.remoteName` is undefined in the typical local desktop use case, so I don't think this will hit the `web` case.

I think we may want to ensure the `local` condition is last, and then add a comment noting nothing should be added after it (Except the `unknown` case)

Reply:
thanks for mentioning that :) pushed a fix.

"local" is 2nd-to-last, since "remote" must be last.